libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/ElapsedTimeTest.java

```diff
@@ -8,19 +8,25 @@
 
 public class ElapsedTimeTest {
 
+  private static final double DELTA = 1E-10;
+
   @Test(expected = NavigationException.class)
   public void errorThrownIfEndCalledBeforeStart() {
     new ElapsedTime().end();
   }
 
   @Test
-  public void elapsedTime() {
+  public void elapsedTime_returnsCorrectTimeInSeconds() {
```

Comment:
Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert

Reply:
This more or less follows arrange, act, assert. I think it's OK as-is.

Reply:
The comment below https://github.com/mapbox/mapbox-navigation-android/pull/1731/files#r255626653 was suppose to be a follow up of this discussion, sorry 😬